### DISCUSSION TITLE
How it is going with a pipeline?

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#19] [General]
How it is going with pipeline? Do you need any help?

**Comment by razvangoga** [comment]
Hi! sorry for the late reply - real life and work got in the way

I manged to get the pipeline working (see here for details  https://github.com/tye-v2/tye2/issues/7#issuecomment-1967567401) and opened a PR for it here : https://github.com/tye-v2/tye2/pull/20.

Let's leave it like this for now so we can move forward with the upgrade to dotnet 8 and the nuget tool publication.

I'll open a different issue for the 6 tests that still fail, as they need some changes in the internal workings of the docker runner class.

**Reply by drakon660** [reply]
No worries, I didn't mind to push you. Lets close this one.

**Reply by razvangoga** [reply]
then review and comment on / approve the PR - it's not really that big - most of the changes are just file moving
